src/core/processing/processingalgorithmparametersmodel.cpp

```diff
@@ -151,8 +151,11 @@ void ProcessingAlgorithmParametersModelBase::rebuild()
           mHasAdvancedParameters = true;
         }
 
-        mParameters << definition;
-        mValues << definition->defaultValue();
+        if ( definition->name() != "INPUT" )
```

Comment:
Just to be extra safe, I've added a definition->type() == QStringLiteral( "source" ) condition here:

https://github.com/opengisch/QField/pull/5690/files#diff-15f7c96b0c450067ace3ed62a83f37fea47213c6cdefbf28bba20747ff6634d2R154